build.ps1

```diff
@@ -35,7 +35,7 @@ if ($RestoreOnly)
 else
 {
     # Build
-    dotnet build -o "$BuildRelativePath" -c $BuildConfig "$Solution" | Tee-Object -FilePath "$LogDirectory\build.log"
+    dotnet build -o "$BuildRelativePath" -c $BuildConfig "$Solution" /p:OutDir=$BuildRelativePath | Tee-Object -FilePath "$LogDirectory\build.log"
```

Comment:
The command `dotnet build` already sets `OutDir` as the value of `-o` parameter. Do we still need `/p:OutDir=$BuildRelativePath`?

Reply:
It turns out that we are fine without specifying the `-o`. So we can remove that. We use the `OutDir` property in the pack command to say where the binaries are.

Reply:
Updated to remove -o.